inc/domain.class.php

```diff
@@ -41,9 +41,757 @@ class Domain extends CommonDropdown {
+         'datatype'           => 'number'
+      ];
+
+      $tab[] = [
+         'id'                 => '80',
+         'table'              => 'glpi_entities',
+         'field'              => 'completename',
+         'name'               => __('Entity'),
+         'datatype'           => 'dropdown'
+      ];
+
+      $tab[] = [
+         'id'                 => '81',
+         'table'              => 'glpi_entities',
+         'field'              => 'entities_id',
+         'name'               => __('Entity-ID')
+      ];
+
+      return $tab;
+   }
+
+   public static function rawSearchOptionsToAdd($itemtype = null) {
+      $tab = [];
+
+      if (in_array($itemtype, Domain::getTypes(true))) {
+         if (Session::haveRight("domain", READ)) {
+            $tab[] = [
+               'id'                 => 'domain',
+               'name'               => self::getTypeName(Session::getPluralNumber())
+            ];
+
+            $tab[] = [
+               'id'                 => '205',
+               'table'              => Domain::getTable(),
+               'field'              => 'name',
+               'name'               => __('Name'),
+               'forcegroupby'       => true,
+               'datatype'           => 'itemlink',
+               'itemlink_type'      => 'Domain',
+               'massiveaction'      => false,
+               'joinparams'         => [
+                  'beforejoin' => [
+                     'table'      => Domain_Item::getTable(),
+                     'joinparams' => ['jointype' => 'itemtype_item']
+                  ]
+               ]
+            ];
+
+            $tab[] = [
+               'id'                 => '206',
+               'table'              => DomainType::getTable(),
+               'field'              => 'name',
+               'name'               => DomainType::getTypeName(1),
+               'forcegroupby'       => true,
+               'datatype'           => 'dropdown',
+               'massiveaction'      => false,
+               'joinparams'         => [
+                  'beforejoin' => [
+                     'table'      => Domain::getTable(),
+                     'joinparams'         => [
+                        'beforejoin' => [
+                           'table'      => Domain_Item::getTable(),
+                           'joinparams' => ['jointype' => 'itemtype_item']
+                        ]
+                     ]
+                  ]
+               ]
+            ];
+         }
+      }
+
+      return $tab;
+   }
+
+   function defineTabs($options = []) {
+      $ong = [];
+      $this->addDefaultFormTab($ong);
+      $this->addStandardTab('DomainRecord', $ong, $options);
+      $this->addStandardTab('Domain_Item', $ong, $options);
+      $this->addStandardTab('Infocom', $ong, $options);
+      $this->addStandardTab('Ticket', $ong, $options);
+      $this->addStandardTab('Item_Problem', $ong, $options);
```

Comment:
I would also add Change tab.
```suggestion
      $this->addStandardTab('Item_Problem', $ong, $options);
      $this->addStandardTab('Change_Item', $ong, $options);
```